jamietre/ImageMapster
# Restrict area highlight to single area and fix USA example

Resolves #332 - Ensure area highlight is restricted to single area at a time
Resolves #333 - Fix USA example for highlighting & selecting area(s)

**Test plan (required)**
In USA.html example:
1) Click State label in right-hand side list - Area should highlight
2) Click another State label in right-hand side list - Area should highlight and previous area should no longer be highlighted
3) Click checkbox of State in right-hand side list - Area should become selected
4) Click checkbox of another State in right-hand side list - Area should become selected and previous area should remain selected
